widget/tree.go

```diff
@@ -766,6 +766,9 @@ func (r *treeContentRenderer) Objects() []fyne.CanvasObject {
 
 func (r *treeContentRenderer) Refresh() {
 	r.refreshForID(r.treeContent.nextRefreshID)
+	for _, s := range r.separators {
```

Comment:
Why was this needed? I think this will lead to RefreshItem, and maybe scrolling, refreshing all separators always, which shouldn't be needed.

Reply:
Without it the resize after changing theme was picking up bad separators.

It seemed like a logical change because the `Refresh` of the content was being called and the items were being refreshed - so shouldn't the separators?

Reply:
(You can see that List has essentially the same - when the content `Refresh()` is called we loop through the separators and refresh them)

Reply:
OK - may be something that can be optimized later since it should only be needed for a full Refresh, and not RefreshItem, etc which will still go through this path. But as you point out it's the same as List for now

Reply:
Good point. I'm not too worried as a separator refresh is so minimal :)

Reply:
I think we should open an issue to make sure we don't forget to do that optimization in the future